tests/phpunit/tests/kses.php

```diff
@@ -1321,6 +1322,20 @@ public function data_safecss_filter_attr() {
 				'css'      => 'filter: url( my-file.svg#svg-blur );',
 				'expected' => 'filter: url( my-file.svg#svg-blur )',
 			),
+			// Support for `repeat` function.
+			array(
+				'css'      => 'grid-template-columns: repeat(4, minmax(0, 1fr))',
+				'expected' => 'grid-template-columns: repeat(4, minmax(0, 1fr))',
+			),
+			array(
+				'css'      => 'grid-template-columns: repeat(auto-fill, minmax(min(12rem, 100%), 1fr))',
+				'expected' => 'grid-template-columns: repeat(auto-fill, minmax(min(12rem, 100%), 1fr))',
+			),
+			// Malformed repeat, no closing `)`.
+			array(
+				'css'      => 'grid-template-columns: repeat(4, minmax(0, 1fr)',
+				'expected' => '',
+			),
```

Comment:
This is me just been paranoid.

```suggestion
			),
			// Malformed repeat, contains unsupported function.
			array(
				'css'      => 'grid-template-columns: repeat(4, unsupported(0, 1fr)',
				'expected' => '',
			),
```

Reply:
looks good! Would it be worth adding a test for an unsupported function within a properly-formed repeat?